doc/api/process.md

```diff
@@ -1687,43 +1702,73 @@ console.log(data);
 
 Additional documentation is available in the [report documentation][].
 
-### process.report.setOptions([options]);
+## process.report.filename
 <!-- YAML
-added: v11.8.0
+added: REPLACEME
 -->
 
-* `options` {Object}
-  * `events` {string[]}
-    * `signal`: Generate a report in response to a signal raised on the process.
-    * `exception`: Generate a report on unhandled exceptions.
-    * `fatalerror`: Generate a report on internal fault
-      (such as out of memory errors or native assertions).
-  * `signal` {string} Sets or resets the signal for report generation
-    (not supported on Windows). **Default:** `'SIGUSR2'`.
-  * `filename` {string} Name of the file where the report is written.
-  * `path` {string} Directory where the report is written.
-    **Default:** the current working directory of the Node.js process.
+* {string}
 
-Configures the diagnostic reporting behavior. Upon invocation, the runtime
-is reconfigured to generate reports based on `options`. Several usage examples
-are shown below.
+Filename where the report is written. The default value is the empty string.
```

Comment:
Probably should document, or point to, the default filename pattern if this is empty.